support-frontend/assets/components/subscriptionCheckouts/billingPeriodSelector.jsx

```diff
@@ -42,26 +43,28 @@ function BillingPeriodSelector(props: PropTypes) {
             billingPeriod === SixWeekly ? Quarterly : billingPeriod, // for 6 for 6 we need the quarterly pricing
             props.fulfilmentOption,
           );
-
-          return (<RadioInputWithHelper
-            text={billingPeriodTitle(billingPeriod)}
-            helper={getPriceDescription(
-              extendedGlyphForCountry(props.billingCountry),
-              productPrice,
-              billingPeriod,
-            )}
-            offer={getAppliedPromoDescription(billingPeriod, productPrice)}
-            name="billingPeriod"
-            checked={billingPeriod === props.selected}
-            onChange={() => props.onChange(billingPeriod)}
-          />);
+          return !props.orderIsAGift || billingPeriod !== SixWeekly ?
```

Comment:
Would `props.orderIsAGift && billingPeriod === SixWeekly` be easier to read here?

Reply:
Yes it would. I just don't like having null as the first option! But you're right. I'll change it.